lib/src/main/res/values/strings.xml

```diff
@@ -1,44 +1,53 @@
+    <string name="token_mismatch_title">Authentication error</string>
     <string name="token_mismatch_message">Provided authentication token does not match or package name is not permitted to perform this action</string>
 
-    <string name="nextcloud_files_app_not_installed_title">Error</string>
-    <string name="nextcloud_files_app_not_installed_message" formatted="true">Nextcloud Files app is not installed. Please install it: %1$s</string>
+    <string name="nextcloud_files_app_not_installed_title">Nextcloud Files not installed</string>
+    <string name="nextcloud_files_app_not_installed_message">This app depends on the Nextcloud Files app. Please install it in order to use this app.</string>
+    <string name="nextcloud_files_app_not_installed_action">Install</string>
 
-    <string name="nextcloud_files_app_account_not_found_title">Error</string>
-    <string name="nextcloud_files_app_account_not_found_message">Account not found in Nextcloud Files app</string>
+    <string name="nextcloud_files_app_account_not_found_title">Account not found</string>
+    <string name="nextcloud_files_app_account_not_found_message">The requested account was not found in Nextcloud Files app</string>
+    <string name="nextcloud_files_app_account_not_found_with_account_message" formatted="true">The account %1$s was not found in Nextcloud Files app</string>
 
-    <string name="nextcloud_files_app_account_permission_not_granted_title">Error</string>
-    <string name="nextcloud_files_app_account_permission_not_granted_message">You have to accept the requested permissions to use the single sign-on feature.</string>
+    <string name="nextcloud_files_app_account_permission_not_granted_title">Permission not granted</string>
+    <string name="nextcloud_files_app_account_permission_not_granted_message">You have to accept the requested permissions to use the Single Sign On feature.</string>
 
-    <string name="nextcloud_http_request_failed_title">Error</string>
+    <string name="nextcloud_http_request_failed_title">Request failed</string>
     <string name="nextcloud_http_request_failed_message" formatted="true">HTTP request failed with HTTP status-code: %1$d</string>
 
     <string name="nextcloud_unsupported_method_title" translatable="false">Error</string>
-    <string name="nextcloud_unsupported_method_message" translatable="false">The requested HTTP Method is not supported yet! Please contact the developers of the Android Single-On Library for Android</string>
+    <string name="nextcloud_unsupported_method_message" translatable="false">The requested HTTP Method is not supported yet! Please contact the developers of the Single Sign On Library for Android</string>
+    <string name="nextcloud_unsupported_method_action">Contact</string>
 
-    <string name="nextcloud_invalid_request_url_title" translatable="false">Error</string>
+    <string name="nextcloud_invalid_request_url_title" translatable="false">Invalid URL</string>
```

Comment:
We generally say "link" instead of "URL" if it's user-facing. Or is this for devs?

Reply:
Yeah, it's more directed to devs. It's not a link which a user is supposed to open in the browser (like `cloud.nextcloud.com`).
It's an URL to an API endpoint, so I'd rather keep the term `URL` here.